### ISSUE TITLE
Missing digits from User accounts overview

### ISSUE THREAD

**Issue by liviuconcioiu** [phpmyadmin/phpmyadmin#18679]
### Describe the bug

On `User accounts overview` page, for digits, if there is no user with a starting digit, that particular digit isn't shown.

This is not the case for letters. If there is no user starting with that letter, the letter is disabled.

### To Reproduce

1. Go to https://demo.phpmyadmin.net/QA_5_2/index.php?route=/server/privileges&viewing_mode=server&server=1
2. Create users with different letters and digits so the pagination will show
3. See missing digits instead of visible as disabled

### Expected behavior

If there's no user with a starting digit, the digit should show as disabled, as is the case with letters.

### Screenshots

![a](https://github.com/phpmyadmin/phpmyadmin/assets/25424343/6f719b61-e7ab-42b2-9ac7-846be7612e4c)

### Server configuration

- phpMyAdmin version: 5.2.2-dev, 6.0.0-dev

**Comment by MoonE** [comment]
I guess this is intentional. Presumably user names can contain pretty much any character. Listing all  possible characters doesn't make sense, so by default the letters A-Z are listed plus any other characters that are in use.

**Comment by williamdes** [comment]
> If there's no user with a starting digit, the digit should show as disabled, as is the case with letters.
I agree, and the last that catches the other cases should stay enabled if there is some
